src/include/sof/math/base2log.h

```diff
@@ -0,0 +1,24 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ *
+ * Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+ *
+ */
+
+#ifndef __SOF_MATH_BASE_TWO_LOGARITHM_H__
+#define __SOF_MATH_BASE_TWO_LOGARITHM_H__
+
+/* Include Files */
+#include <stdint.h>
+
+/* Defines Constant*/
+#define BASE2LOG_WRAP_SCHAR_BITS 0xFF
+#define BASE2LOG_UPPERBYTES 0xFFFFFF
+#define BASE2LOG_WORDLENGTH 0x1F
+
+/* Function Declarations */
+int32_t base2_logarithm(uint32_t u);
+void testvector(uint32_t u[100]);
```

Comment:
This function prototype looks unnecessary. I don't see use for testvector().

Reply:
I will removed redundant function call